src/SonataCommentBundle.php

```diff
@@ -51,16 +44,18 @@ public function boot()
     /**
      * Register form mapping information.
      */
-    public function registerFormMapping()
+    private function registerFormMapping()
     {
-        FormHelper::registerFormTypeMapping([
-            'fos_comment_comment' => 'FOS\CommentBundle\Form\CommentType',
-            'fos_comment_commentable_thread' => 'FOS\CommentBundle\Form\CommentableThreadType',
-            'fos_comment_delete_comment' => 'FOS\CommentBundle\Form\DeleteCommentType',
-            'fos_comment_thread' => 'FOS\CommentBundle\Form\ThreadType',
-            'fos_comment_vote' => 'FOS\CommentBundle\Form\VoteType',
-            'sonata_comment_comment' => 'Sonata\CommentBundle\Form\Type\CommentType',
-            'sonata_comment_status' => 'Sonata\CommentBundle\Form\Type\CommentStatusType',
-        ]);
+        if (class_exists('Sonata\CoreBundle\Form\FormHelper')) {
+            \Sonata\CoreBundle\Form\FormHelper::registerFormTypeMapping([
```

Comment:
You can still have it in a use statement, please revert that change.

Reply:
Can i change this methods to private? It is potentialy BC-break but i can't imagine case when someone using it or extending.

Reply:
It's pretty unlikely, but do you _need_ to?